controllers/session/conditions.go

```diff
@@ -10,16 +10,17 @@ import (
 )
 
 func createConditionForLocatedRef(ref model.Ref, located model.LocatorStatus) istiov1alpha1.Condition {
-	message := located.Kind + "/" + located.Name + " status " + ref.KindName.String() + ": "
+	message := located.GetNamespaceName() + "[" + located.Kind + "] status " + ref.KindName.String() + ": "
```

Comment:
I thought this way it's easier to understand when going through `Condition`s in the `Session` object.